tests/pravega_client_test.py

```diff
@@ -57,6 +58,23 @@ def test_tags(self):
         tags = stream_manager.get_stream_tags(scope, "testStream1")
         self.assertEqual(["t4", "t5"], tags)
 
+        # test for list_scope api
+        scope_list = stream_manager.list_scope()
+        self.assertTrue(len(scope_list) > 0, "The list is empty")
```

Comment:
Instead of scope_list > 0, can we update the assert to length equal to 1.

Reply:
As we checked offline, it can't be updated with length equal to 1, as it retains all the scope that exist in standalone run too.
So it may or may not be always 1.